evap/staff/importers.py

```diff
@@ -151,6 +152,16 @@ def get_course_or_add_error(self, course_type_name, errors):
             errors['course_type'] = course_type_name
             return None
 
+    def parse_is_graded_or_add_error(self, is_graded, errors):
+        is_graded = is_graded.strip()
+        if is_graded == settings.IMPORTER_GRADED_YES:
+            return True
+        elif is_graded == settings.IMPORTER_GRADED_NO:
+            return False
+        else:
+            errors['is_graded'] = is_graded
+            return None
```

Comment:
Hmm, funny that pylint doesn't complain here, I thought we had a rule about if-return-else (will have to look into that, I changed about 50 occurences to be consistent, I was sure there was a rule that failed)

Could you make it either
```python
if is_graded == settings.IMPORTER_GRADED_YES:
    return True

if is_graded == settings.IMPORTER_GRADED_NO:
    return False

errors['is_graded'] = is_graded
return None
```


or

````python
if is_graded not in [settings.IMPORTER_GRADED_YES, settings.IMPORTER_GRADED_NO]:
    errors['is_graded'] = is_graded
    return None

return is_graded == settings.IMPORTER_GRADED_YES
````

for consistency with the rest of the code base?

Reply:
Just noticed that you just moved it. In this case, I'm also fine with leaving it as it was.

Reply:
Hmm, so pylint has this rule `no-else-return` (R1705), we haven't disabled it, yet it doesn't complain here...

Well, I guess if it doesn't complain, neither do it.

Reply:
Indeed, during development pylint has complained about this piece of code (one for `elif`, one for `else`). I assigned the fix for this to the wrong commit previously, sorry for confusing you

Reply:
ah that makes sense and also explains why the comment was instantly outdated :D